test/preview/document_stores/_base.py

```diff
@@ -64,17 +78,29 @@ def test_no_filter_not_empty(self, docstore):
         assert docstore.filter_documents() == docs
         assert docstore.filter_documents(filters={}) == docs
 
-    def test_filter_simple_value(self, docstore, filterable_docs):
+    def test_filter_simple_metadata_value(self, docstore, filterable_docs):
         self.direct_write(docstore, filterable_docs)
-        result = docstore.filter_documents(filters={"year": "2020"})
+        result = docstore.filter_documents(filters={"page": "100"})
         assert len(result) == 3
 
+    def test_filter_document_content(self, docstore, filterable_docs):
+        self.direct_write(docstore, filterable_docs)
```

Comment:
I'm also not too happy about it but I could not come up with a better way other than using `docstore.write()`. Shall I just do that? It would seem to me like I'm testing unrelated things. Or should I provide a fixture with a pre-populated document store? I think this one is a slippery slope (how many versions of pre-populated docstores we'll end up having?) etc etc...

Reply:
Not sure why we don't just go with `docstore.write_documents()`, to me it looks like any other method you have to call on that class to make it ready to be tested, same as (let's say) `__init__`